### PR TITLE
Prepared bugfix release v0.15.2

### PR DESCRIPTION
**Pull request by naveen521kk**
## Overview: What does this pull request change?
Changelog and bumped version number for bugfix release.

## Links to added or changed documentation pages

https://manimce--2694.org.readthedocs.build/en/2694/changelog/0.15.2-changelog.html

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by naveen521kk** [comment]
@behackl in https://github.com/ManimCommunity/manim/pull/2596/files#diff-58a1af5ffb97c92dd50c8d314f21ea64969177aa9d6ba9261e8003a26fef8a27R75 the release PR was also added to changelog while it isn't in this one. Am I missing some step?

**Review by Kiran-Raj-Dev — APPROVED** [review]
LGTM, Thanks :)

File: docs/source/changelog/0.15.2-changelog.rst [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+* :pr:`2656`: Fixed incompatibility with black version
+
+
+Code quality improvements and similar refactors
+-----------------------------------------------
+
+* :pr:`2630`: Remove WebGL renderer
+   The WebGL renderer is broken and is unmaintained anymore. The support for it in Manim is removed.
+
+* :pr:`2652`: Update cloup version to 0.13.0 from 0.7.0
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:187]:
```suggestion
* :pr:`2652`: Update ``cloup`` version to 0.13.0 from 0.7.0
```

File: docs/source/changelog/0.15.2-changelog.rst [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+
+* :pr:`2656`: Fixed incompatibility with black version
+
+
+Code quality improvements and similar refactors
+-----------------------------------------------
+
+* :pr:`2630`: Remove WebGL renderer
+   The WebGL renderer is broken and is unmaintained anymore. The support for it in Manim is removed.
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:185]:
```suggestion
   The WebGL renderer is broken and unmaintained. The support for it in Manim is removed.
```

File: docs/source/changelog/0.15.2-changelog.rst [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+* :pr:`2685`: Migrate from os.path to pathlib in Testing Scripts
+   This pull request changes a number of instances of ``os.path`` to Pathlib objects and functions. In addition, this PR modifies the SVGMobject constructor to accept both a Pathlib object or a string variable pathname its constructor.
+
+* :pr:`2691`:  Removed :class:`CameraFrame`
+
+
+* :pr:`2696`: Improve ``scripts/dev_changelog.py``
+
+
+* :pr:`2697`: devchangelog: Sort items before writing
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:202]:
```suggestion
* :pr:`2697`: Sort PRs by number in changelog sections before writing
```

File: docs/source/changelog/0.15.2-changelog.rst [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+* :pr:`2678`: Require ``backports-cached-property`` only for Python < 3.8
+
+
+* :pr:`2685`: Migrate from os.path to pathlib in Testing Scripts
+   This pull request changes a number of instances of ``os.path`` to Pathlib objects and functions. In addition, this PR modifies the SVGMobject constructor to accept both a Pathlib object or a string variable pathname its constructor.
+
+* :pr:`2691`:  Removed :class:`CameraFrame`
+
+
+* :pr:`2696`: Improve ``scripts/dev_changelog.py``
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:199]:
```suggestion
* :pr:`2696`: Made changelog generation run in parallel plus further improvements to ``scripts/dev_changelog.py``
```

File: docs/source/changelog/0.15.2-changelog.rst [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+
+* :pr:`2640`: Copyedited Documentation
+   Reviewed ``tutorials/configurations.rst``. Edited simple mistakes such as Manim not being capitalized and commas.
+
+* :pr:`2649`: Document and type utils/iterables.py
+
+
+* :pr:`2651`: Update copyright year in documentation to 2020-2022
+   Updates year in ``docs/conf.py`` to correct year in the footer of the documentation.
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:164]:
```suggestion
```

File: docs/source/changelog/0.15.2-changelog.rst [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+* :pr:`2676`: Fixed opengl finish transform
+
+
+* :pr:`2688`: Fixed two minor issues of :class:`.SpiralIn` and :class:`.ManimBanner`
+
+
+Documentation-related changes
+-----------------------------
+
+* :pr:`2609`: Copy Edit troubleshooting.rst
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:142]:
```suggestion
* :pr:`2609`: Copyedit troubleshooting.rst
```

File: docs/source/changelog/0.15.2-changelog.rst [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+* :pr:`2648`: Handle user-defined centers for Wiggle animation
+
+
+* :pr:`2658`: Fix arguments of overridden ``set_style`` for :class:`.BackgroundRectangle`
+   Using :class:`.Write` animation on a :class:`.Text` object with ``.add_background_rectangle()`` applied no longer generates a ``TypeError``.
+
+* :pr:`2668`: (Re)set background color of :class:`.OpenGLRenderer` when initializing scene
+
+
+* :pr:`2676`: Fixed opengl finish transform
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:133]:
```suggestion
* :pr:`2676`: Fixed propagation of custom attributes in animations for the OpenGL renderer
```

File: docs/source/changelog/0.15.2-changelog.rst [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+   - Add documentation for ``simple_functions.py``.
+   - Small additions with some extra clarity for these functions.
+
+* :pr:`2693`: Allow using :meth:`.MovingCamera.auto_zoom` without animation
+   Allows auto zooming camera without having to play an animation by passing an ``animation=False`` argument
+
+Fixed bugs
+----------
+
+* :pr:`2546`: Fixed a File logging bug and some maintenance
```
Comment (behackl) [inline docs/source/changelog/0.15.2-changelog.rst:110]:
```suggestion
* :pr:`2546`: Fixed a file logging bug and some maintenance
```

**Review by behackl — APPROVED** [review]
LGTM, suggested some minor improvements that could simply be batch-commited.

**Comment by naveen521kk** [comment]
Thanks for the reviews!

**Pull request merged**